### DISCUSSION TITLE
Example not working (for me)

### DISCUSSION THREAD

**Discussion by Noml** [ManimCommunity/manim#1420] [Q&A]
This is definitely just my computer, but the examples for the [TexTransform](https://3b1b.github.io/manim/getting_started/example_scenes.html#textransformexample) and [TransformMatchingTex](https://docs.manim.community/en/stable/reference/manim.animation.transform_matching_parts.TransformMatchingTex.html#manim.animation.transform_matching_parts.TransformMatchingTex) are not working for me. When I go to run the program, (after copying and pasting it) everything else runs fine except it says:

> NameError: name 'TransformMatchingShapes' is not defined 
> NameError: name 'TransformMatchingTex' is not defined

Other MathTex and Tex things work fine, but I can't get these to work. Any tips?

**Comment by BeeGass** [comment]
I had a very similar situation but `Create` had not been defined. I went back through the installation and made sure that all dependencies were installed. You may have done the installation process correctly but its always good to isolate the issue. 

I believe the way I fixed it was by re-entering `choco install manimce`

EDIT:
I lied to you, the issue is that apparently the pip installs install a much older version of manim. In order to get the functions that they describe in the tutorials and the ones you are trying to use I believe you need to clone from the repository. For reference I used [this video](https://www.youtube.com/watch?v=Upztg_Bs-zs) to help me with the installation process. PS, I wouldnt follow the virtual environment stuff unless you absolutely want to.

**Reply by behackl** [reply]
> the issue is that apparently the pip installs install a much older version of manim

pip should install the latest released version of Manim (currently: `v0.6.0`) by running `pip install manim` -- and for upgrading, `pip install -U manim`.

(Note that the package is actually called `manim`, in contrast to `manimce` for the chocolatey install.)

**Comment by behackl** [comment]
How did you install Manim? To use these classes you'll need a version of v0.5.0 or later.

**Comment by Darylgolden** [comment]
To add on, you can update manim by typing `pip install -U manim` in your terminal.
